meta/group-info/de.yaml

```diff
@@ -0,0 +1,67 @@
+separator: ", "
+alumni:
+  roles:
+    academy: osu!academy-Gründer
+    admin: Administration
+    chat: Chat Moderator
+    dev: osu!dev
+    media: osu!media Erstellung
+    support: Support-Team
+    tournaments: Verwaltung von Wettbewerben
+gmt:
+  all_mods: alle Moderatoren
+  areas:
+    beatmap: Beatmap-Moderation
+    chat: Chat-Moderation
+    forum: Forum-Moderation
+    team: offizielle Präsenz des Teams
+    dev: osu! Entwicklung
+    player: Spieler-Support
+    support: Technischer Support
+    tournaments: Verwaltung von Wettbewerben
+    wiki: Wiki-Administration
+    mapping: Moderation der Mapping/Modding Community
+nat:
+  areas:
+    communication: Kommunikation
+    evaluation: Beurteilung
+    moderation: Moderation
+    structural: strukturell
```

Comment:
I've looked at the `NAT` article and there seems to be only one case where this needs to have another grammar form and that is the text which describes what this word actually means. I guess the mention in the text can be translated manually. The only thing that I would change is `Strukturell` instead of `strukturell`. You may want to take a look at the article yourself (`People\The_Team\Nomination_Assessment_Team`).

Reply:
Agree, I'll capatalize it.